cmd/process-agent/api/server.go

```diff
@@ -18,7 +18,8 @@ import (
 )
 
 func setupHandlers(r *mux.Router) {
-	r.HandleFunc("/config", settingshttp.Server.GetFull("process_config")).Methods("GET")
+	r.HandleFunc("/config", settingshttp.Server.GetFull("process_config")).Methods("GET") // Get only settings in the process_config namespace
+	r.HandleFunc("/config/all", settingshttp.Server.GetFull("")).Methods("GET")           // Get all fields from process-agent Config object
```

Comment:
Nit: seems overkill to distinguish between process-agent only vs not and there could be misconfigurations where the process agent has common settings set on its container. But if we find this useful let's keep it.

Reply:
I agree with you. I've decided to keep the `/config` endpoint fetching only `process_config` settings because it's used by the `config` command and I didn't want to change its behavior.
I'll keep both of them for now and we can revisit it later, what do you think?